src/dist/linear_regression.cpp

```diff
@@ -17,27 +17,33 @@ std::tuple<float, float> fit_slope(const arma::mat& kmers,
                                    Reference * r1,
                                    Reference * r2) {
 
+    // Store core/accessory in dists, truncating at zero
+    float core_dist = 0, accessory_dist = 0;
+    static const double tolerance = (5.0 / (r1->sketchsize64() * 64));
```

Comment:
Just curious why you changed the tolerance threshold from your original proposal?

Reply:
I have forgotten what my original proposal was?

Reply:
From Teams: Nothing below log(1/s) makes sense, but probably counts around that size are all problematic - don't know if that rings a bell, I guess this is more conservative?

Reply:
That makes sense - I didn't understand the need to log out of context previously.